deisctl/units/deis-controller.service

```diff
@@ -6,8 +6,10 @@ After=deis-store-volume.service
 [Service]
 EnvironmentFile=/etc/environment
 TimeoutStartSec=20m
+ExecStartPre=-/usr/bin/etcdctl mkdir /deis/scheduler >/dev/null 2>&1
```

Comment:
> if it failed due to etcd errors, don't we want to exit there?

Ideally yes, but as you pointed out it's hard to see a clean way to do that. Maybe in this case we have to allow `confd` to report the problem later. :disappointed: